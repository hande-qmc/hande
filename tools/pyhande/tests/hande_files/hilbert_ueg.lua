-- Create output with:
-- $[HANDE DIR]/bin/hande.x hilbert_ueg.lua > hilbert_ueg.out 2> hilbert_ueg.err
-- Note that these settings are just for testing...
sys = ueg {
    dim = 3,
    nel = 14,
    ms = 0,
    cutoff = 1,
}

hilbert_space {
    sys = sys,
    hilbert = {
        rng_seed = -563090706,
        nattempts = 1000,
        ncycles = 3,
    },
}
