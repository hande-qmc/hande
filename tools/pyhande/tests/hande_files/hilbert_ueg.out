
   HANDE

 ================================================================
Version information
-------------------
Version       | v1.4-25-g4efb802
Commit hash   | 4efb80270c639ebaf202fe7b326e94a9ccfefafe
Commit author | Verena Neufeld
Commit date   | Thu Mar 21 12:08:44 2019 +0000
Branch        | master

Configuration and build information
-----------------------------------
Who compiled             | verena
Compilation hostname     | verena-Think
Operating system         | Linux-4.4.0-176-generic
CMake version            | 3.10.2
CMake generator          | Unix Makefiles
Build type               | release
Configuration time       | 2020-03-31 15:44:09 [UTC]
Python version           | 3.6.10
Fortran compiler         | /usr/bin/gfortran
C compiler               | /usr/bin/gcc
DET_SIZE set to          | 64
POP_SIZE set to          | 64
dSFMT Mersenne exponent  | 19937
MPI parallelization      | OFF
OpenMP parallelization   | OFF
Lua version              | 5.3.5
HDF5 library version     | 1.10.2

Further components
------------------
DISABLE_HDF5 not defined.  HDF5 enabled.
DISABLE_LANCZOS defined.  Lanczos disabled.
DISABLE_UUID not defined.  UUID enabled.
PARALLEL not defined.  MPI parallelization disabled.
DISABLE_SCALAPACK not defined.  ScaLAPACK enabled.
SINGLE_PRECISION not defined.  Double precision used throughout.
USE_POPCNT not defined. Internal POPCNT procedure used.

Runtime information
-------------------
Hostname:
    verena-Think
Current working dir:
    /home/verena/code3/hande/tools/pyhande/pyhande/tests/hande_files
Started running on 01/04/2020 at 20:45:21
 Calculation UUID: 6c228dba-68c9-4051-b47a-5704fe261ad8.
 ================================================================

 Input options
 -------------

-- Create output with:
-- $[HANDE DIR]/bin/hande.x hilbert_ueg.lua > hilbert_ueg.out 2> hilbert_ueg.err
-- Note that these settings are just for testing...
sys = ueg {
    dim = 3,
    nel = 14,
    ms = 0,
    cutoff = 1,
}

hilbert_space {
    sys = sys,
    hilbert = {
        rng_seed = -563090706,
        nattempts = 1000,
        ncycles = 3,
    },
}

 -------------

 Basis functions
 ---------------

 Spin given in units of 1/2.
 k-points given in terms of the reciprocal lattice vectors of the crystal cell.

 index   k-point            ms       <i|h|i>
     1   (  0,  0,  0)      1        0.00000000E+00
     2   (  0,  0,  0)     -1        0.00000000E+00
     3   (  0,  0, -1)      1        1.30773168E+00
     4   (  0,  0, -1)     -1        1.30773168E+00
     5   (  0, -1,  0)      1        1.30773168E+00
     6   (  0, -1,  0)     -1        1.30773168E+00
     7   ( -1,  0,  0)      1        1.30773168E+00
     8   ( -1,  0,  0)     -1        1.30773168E+00
     9   (  1,  0,  0)      1        1.30773168E+00
    10   (  1,  0,  0)     -1        1.30773168E+00
    11   (  0,  1,  0)      1        1.30773168E+00
    12   (  0,  1,  0)     -1        1.30773168E+00
    13   (  0,  0,  1)      1        1.30773168E+00
    14   (  0,  0,  1)     -1        1.30773168E+00
    15   (  0, -1, -1)      1        2.61546336E+00
    16   (  0, -1, -1)     -1        2.61546336E+00
    17   ( -1,  0, -1)      1        2.61546336E+00
    18   ( -1,  0, -1)     -1        2.61546336E+00
    19   (  1,  0, -1)      1        2.61546336E+00
    20   (  1,  0, -1)     -1        2.61546336E+00
    21   (  0,  1, -1)      1        2.61546336E+00
    22   (  0,  1, -1)     -1        2.61546336E+00
    23   ( -1, -1,  0)      1        2.61546336E+00
    24   ( -1, -1,  0)     -1        2.61546336E+00
    25   (  1, -1,  0)      1        2.61546336E+00
    26   (  1, -1,  0)     -1        2.61546336E+00
    27   ( -1,  1,  0)      1        2.61546336E+00
    28   ( -1,  1,  0)     -1        2.61546336E+00
    29   (  1,  1,  0)      1        2.61546336E+00
    30   (  1,  1,  0)     -1        2.61546336E+00
    31   (  0, -1,  1)      1        2.61546336E+00
    32   (  0, -1,  1)     -1        2.61546336E+00
    33   ( -1,  0,  1)      1        2.61546336E+00
    34   ( -1,  0,  1)     -1        2.61546336E+00
    35   (  1,  0,  1)      1        2.61546336E+00
    36   (  1,  0,  1)     -1        2.61546336E+00
    37   (  0,  1,  1)      1        2.61546336E+00
    38   (  0,  1,  1)     -1        2.61546336E+00

 Number of electrons: 14
 Number of basis functions: 38

 Bit-length of integers used to store determinant bit-strings: 64
 Number of integers used to store determinant bit-strings: 1

 Hilbert space
 -------------

 -- Start JSON block --
 {
     "system": {
         "nbasis": 38,   
         "nel": 14,   
         "nvirt": 24,   
         "Ms": 0,   
         "nalpha": 7,   
         "nbeta": 7,   
         "nvirt_alpha": 12,   
         "nvirt_beta": 12,   
         "nsym": 19,   
         "sym0": 1,   
         "sym_max": 19,   
         "nsym_tot": 19,   
         "sym0_tot": 1,   
         "sym_max_tot": 19,   
         "symmetry": 2147483647,   
         "tot_sym": false,   
         "aufbau_sym": true,   
         "max_number_excitations": 14,   
         "ueg": {
             "r_s": 1.00000000,   
             "ecutoff": 1.00000000,   
             "k_fermi": 1.91915829,   
             "E_fermi": 1.84158428,   
             "ktwist": [0.00000000,0.00000000,0.00000000],
             "L": [3.88512994,3.88512994,3.88512994]
         }
     },
     "ex_level": 14,   
     "nattempts": 1000,   
     "ncycles": 3,   
     "occ_list": [],
     "rng_seed": -563090706    
 }
 -- End JSON block --


 Reference determinant and so symmetry sector selected using the Aufbau principle.
 Selected symmetry  1.

 Symmetry of reference determinant: (  0,  0,  0)

 space size: estimate of the Hilbert space size from a single iteration.
 mean: running estimate of the mean of the Hilbert space size.
 std. err.: running estimate of the standard error in the estimate of the mean.

 # iterations  space size    mean          std. err.
            1  1.523370E+07  1.523370E+07           NaN
            2  2.031160E+07  1.777265E+07  2.538951E+06
            3  1.015580E+07  1.523370E+07  2.931728E+06

 Monte-Carlo estimate of size of space is: 1.523370E+07 +/- 2.931728E+06

 Timing breakdown
 ----------------

 Time for each calculation section (seconds):

 UEG initialisation      : .00
 Hilbert space estimation: .00

 ================================================================
 Finished running on 01/04/2020 at 20:45:21
 Wall time (seconds):                           0.00
 CPU time (per processor, seconds):             0.00
 ================================================================

